schemas/enums.yml

```
@@ -0,0 +1,24 @@
+components:
+  schemas:
+    SocialNetworkType:
+      type: string
+      enum:
+        - Youtube
+        - Github
+        - LinkedIn
+        - Instagram
+        - Facebook
+        - X
+        - Medium
+        - Slack
+        - Meetup
+
+    MemberType:
+      type: string
+      enum:
+        - WCC Director
+        - WCC Leader
```

```suggestion
        - Leader
```
As we decided to replace `WCC Director` by `Director`, maybe it is reasonable to do the same for `WCC Leader`. What do you think?

---

Yep, that is the plan.